docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -62,7 +62,34 @@ There you can find links to upgrade notes for other versions too.
     - read the section about proxying the URL content subpaths via webserver domain [`docs/introduction/abstract-filesystem.md`](https://github.com/shopsys/shopsys/blob/master/docs/introduction/abstract-filesystem.md)
 - to be more descriptive about error caused by active TEST environment ([#701](https://github.com/shopsys/shopsys/pull/701))
     - modify `ErrorController::createUnableToResolveDomainResponse()` by these [changes](https://github.com/shopsys/shopsys/pull/701/files#diff-0b1aecbf82624ce474ca3cb8bd75811c).
-
+- start filtering products with elasticsearch ([#943](https://github.com/shopsys/shopsys/pull/943))
```

Comment:
We need split the upgrades to two sections - one for people that want to keep the SQl filter and the other one for the elastic filter